addon/components/frost-text.js

```diff
@@ -60,6 +61,7 @@ export default Component.extend(FrostEventsProxyMixin, {
       isClearEnabled: false,
       isClearVisible: false,
       isHookEmbedded: false,
+      onClear: () => {},
```

Comment:
This does not match the pattern used elsewhere in the repo, such as at https://github.com/ciena-frost/ember-frost-core/blob/master/addon/components/frost-expand.js#L94

Was this deviation intentional?

Reply:
Nope, didn't see the other pattern. I'll change it to match those